style.css

```diff
@@ -32,106 +29,122 @@ hr {
+	grid-template-columns: auto;
  }
 
  /*----------------BURGER NAV---------------------*/
+ 
+ .nav{
+	box-sizing: border-box;
+	background-color: #f7f7f7;
+	padding-left: 1vw;
+	padding-right: 1vw;
+	background: rgba(204, 204, 204, 0.7);
+	grid-column: 1 / span 6;
+	grid-row: 1;
+}
 
 #menu__toggle {
-	  opacity: 0;
+	opacity: 0;
 }
 
 #menu__toggle:checked + .menu__btn > span {
-	  transform: rotate(45deg);
+	transform: rotate(45deg);
 }
 
 #menu__toggle:checked + .menu__btn > span::before {
-	  top: 0;
-	  transform: rotate(0deg);
+	top: 0;
+	transform: rotate(0deg);
 }
 
 #menu__toggle:checked + .menu__btn > span::after {
-	  top: 0;
-	  transform: rotate(90deg);
+	top: 0;
+	transform: rotate(90deg);
 }
 
 #menu__toggle:checked ~ .menu__box {
-	  left: 0 !important;
+	left: 0 !important;
 }
 
 .menu__btn {
-	  position: fixed;
-	  top: 40px;
-	  left: 20px;
-	  width: 26px;
-	  height: 26px;
-	  cursor: pointer;
-	  z-index: 3;
+	position: absolute;
+	top: 2vw;
+	left: 1vw;
+	width: 1.8vw;
+	height: 2vw;
+	cursor: pointer;
+	z-index: 3;
 }
 
 .menu__btn > span,
 .menu__btn > span::before,
 .menu__btn > span::after {
-	  display: block;
-	  position: absolute;
-	  width: 100%;
-	  height: 3px;
-	  background-color: #616161;
-	  transition-duration: .30s;
+	display: block;
+	position: absolute;
+	width: 100%;
+	height: 0.25vw;
+	background-color: #616161;
+	transition-duration: .30s;
 }
 
 .menu__btn > span::before {
-	  content: '';
-	  top: -8px;
+	content: '';
+	top: -8px;
 }
 
 .menu__btn > span::after {
-	  content: '';
-	  top: 8px;
+	content: '';
+	top: 8px;
 }
 
 .menu__box {
-	  display: block;
-	  position: fixed;
-	  top: 0;
-	  left: -100%;
-	  width: 300px;
-	  height: 100%;
-	  margin: 0;
-	  padding: 80px 0;
-	  list-style: none;
-	  background-color: #ECEFF1;
-	  box-shadow: 0px 0px 0px rgba(0, 0, 0, .4);
-	  transition-duration: .30s;
+	display: block;
+	position: absolute;
+	top: 0;
+	left: -100%;
+	width: 13vw;
+	height: 100%;
+	margin: 0;
+	padding: 6vw 0;
+	list-style: none;
+	background-color: #ECEFF1;
+	box-shadow: 0px 0px 0px rgba(0, 0, 0, .4);
+	transition-duration: .30s;
 }
 
 .menu__item {
-	  display: block;
-	  padding: 12px 24px;
-	  color: #333;
-	  font-family: 'Roboto', sans-serif;
-	  font-size: 20px;
-	  font-weight: 600;
-	  text-decoration: none;
-	  transition-duration: .30s;
+	display: block;
+	padding: 1vw 2vw;
+	color: #333;
+	font-family: 'Roboto', sans-serif;
+	font-size: 1.5vw;
+	font-weight: 600;
+	text-decoration: none;
+	transition-duration: .30s;
 }
 
 .menu__item:hover {
-	  background-color: #f7f7f7;
+	background-color: #f7f7f7;
 }
 
  /*-----------------------------------------------*/
  
 .header{
-	grid-column: 2/6;
-	grid-row: 1;
-	/*background-color: #f7f7f7;*/
 	box-sizing: border-box;
-	height: 50vh;	/*view-port height is height of viewing area on screen*/
-	padding-top: 10px;
-	background-size: cover;
-	background-position: top center;
+	background-color: blue;
```

Comment:
I removed bulk of the header as it was not needed. I still left the header class here just in case I will need to use it later.